### PR TITLE
add release notes for v0.9.0-alpha0

### PR DESCRIPTION
**Pull request by fitzthum**
This is an alpha release, so let's be clear about exactly what the limitations are.

### CODE REVIEW

**Review by stevenhorsman — APPROVED** [review]
LGTM and suitably disclaimer-y! Thanks

File: releases/v0.9.0-alpha0.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+definition of the acronyms used in this document.
+
+## What's new
+
+* This release is built from the main branch of Kata containers
+* Non-tee attestation is now based on a sample attester and verifier rather than on `offline_fs_kbc`.
+    * Resources can be dynamically delivered in confidential environments
+    * Trustee is integrated into the Kata CI
+* All platforms now share one confidential rootfs
+* All platforms share one confidential guest kernel
```
Comment (mythi) [inline releases/v0.9.0-alpha0.md:34]:
```suggestion
* All platforms share one confidential guest kernel with the support of `configfs-tsm` report ABI
      * this release adds a module for attester vendors to adopt the new kernel ABI and TDX attester moving to use it by default 
```

File: releases/v0.9.0-alpha0.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+consider returning to v0.8.0 or using the next release.
+
+Please see the [quickstart guide](../quickstart.md) for details on how to try out Confidential
+Containers.
+
+Please refer to our [Acronyms](https://github.com/confidential-containers/documentation/wiki/Acronyms)
+and [Glossary](https://github.com/confidential-containers/documentation/wiki/Glossary) pages for a
+definition of the acronyms used in this document.
+
+## What's new
```
Comment (mythi) [inline releases/v0.9.0-alpha0.md:27]: enclave-cc gained several improvements (I don't mean the dependabot PRs ;) but in one sentence

* enclave-cc moved to a unified LibOS bundle for secure rootfs key handling and to the latest Occlum v0.30.1 release that adds SGX EDMM support for dynamically adjusting the enclave size.

File: releases/v0.9.0-alpha0.md (hunk 1 above)
Comment (fitzthum) [reply releases/v0.9.0-alpha0.md]: I added a new bullet for this.

File: releases/v0.9.0-alpha0.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+* Pulling container images inside guest may have negative performance implications including greater resource usage and slower startup.
+* `crio` support is still evolving.
+* Platform support is rapidly changing
+* SELinux is not supported on the host and must be set to permissive if in use.
+* The format of encrypted container images is still subject to change
+  * The [oci-crypt](https://github.com/containers/ocicrypt) container image format itself may still change
+  * The tools to generate images are not in their final form
+  * The image format itself is subject to change in upcoming releases
+  * Not all image repositories support encrypted container images.
+ Complete integration with Kubernetes is still in progress.
```
Comment (wainersm) [inline releases/v0.9.0-alpha0.md:65]: Missing the bullet marker `*` as `* Complete integration with....`. The format is wrong on v0.8.0 release notes too.

File: releases/v0.9.0-alpha0.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
+# Release Notes for v0.9.0-alpha
+
+Release Date: April 25th, 2024
+
+This is our first release based on Kata main, but it is an alpha release that supports
+only a subset of features.
+
+This release is based on [3.4.0](https://github.com/kata-containers/kata-containers/releases/tag/3.4.0) of Kata
```
Comment (wainersm) [inline releases/v0.9.0-alpha0.md:8]: Hi @fitzthum !

The Kata teams recommends to use full project name "Kata Containers" when possible.

File: releases/v0.9.0-alpha0.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+Please see the [quickstart guide](../quickstart.md) for details on how to try out Confidential
+Containers.
+
+Please refer to our [Acronyms](https://github.com/confidential-containers/documentation/wiki/Acronyms)
+and [Glossary](https://github.com/confidential-containers/documentation/wiki/Glossary) pages for a
+definition of the acronyms used in this document.
+
+## What's new
+
+* This release is built from the main branch of Kata containers.
```
Comment (wainersm) [inline releases/v0.9.0-alpha0.md:29]: s/containers/Containers

**Review by sameo — APPROVED** [review]

File: releases/v0.9.0-alpha0.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+  * The [oci-crypt](https://github.com/containers/ocicrypt) container image format itself may still change
+  * The tools to generate images are not in their final form
+  * The image format itself is subject to change in upcoming releases
+  * Not all image repositories support encrypted container images.
+* Complete integration with Kubernetes is still in progress.
+  * OpenShift support is not yet complete.
+  * Existing APIs do not fully support the CoCo security and threat model. [More info](https://github.com/confidential-containers/community/issues/53)
+  * Some commands accessing confidential data, such as `kubectl exec`, may either fail to work, or incorrectly expose information to the host
+* The CoCo community aspires to adopting open source security best practices, but not all practices are adopted yet.
+  * We track our status with the OpenSSF Best Practices Badge, which improved to 69% at the time of this release.
```
Comment (dcmiddle) [inline releases/v0.9.0-alpha0.md:70]:
```suggestion
  * We track our status with the OpenSSF Best Practices Badge, which improved to 75% at the time of this release.
```

File: releases/v0.9.0-alpha0.md [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+  * Vulnerability reporting mechanisms still need to be created. Public github issues are still appropriate for this release until private reporting is established.
```
Comment (dcmiddle) [inline releases/v0.9.0-alpha0.md:71]:
```suggestion
  * We resolved vulnerability reporting mechanisms and a security policy. Remaining work is to apply and document static and dynamic analysis. 
```

Comment (fitzthum) [reply releases/v0.9.0-alpha0.md]: I added something to the what's new section for this as well.

**Review by magowan — APPROVED** [review]
Thanks Tobin, looks good to me

**Review by dcmiddle — APPROVED** [review]

**Pull request merged**